RFC/src/RFC-0172_PeerToPeerMessagingProtocol.md

```diff
@@ -266,19 +269,19 @@ It consists of a [MessageHeader] and [Message] of a particular predefined [Messa
 
```

Comment:
Please remove square brackets here:
`A MessageEnvelopeBody is the payload of the [MessageEnvelope]. A [MessageEnvelopeBody] may be encrypted as required.` 
to
`A MessageEnvelopeBody is the payload of the MessageEnvelope. A MessageEnvelopeBody may be encrypted as required.`